qiime2/sdk/tests/test_plugin_manager.py

```diff
@@ -86,9 +89,43 @@ def test_importable_types(self):
         self.assertNotIn(Kennel, types)
 
     # TODO: add tests for type/directory/transformer registrations
+    def test_get_formats_include_all_formats(self):
+        obs = self.pm.formats
+        exp = self.pm.get_formats(include_all=True, importable=False,
+                                  exportable=False, canonical_format=False)
+
+        self.assertEqual(obs, exp)
+
+    def test_get_formats_importable_formats(self):
+        obs = self.pm._importable
+        exp = self.pm.get_formats(include_all=False, importable=True,
+                                  exportable=False, canonical_format=False)
+
+        self.assertEqual(obs, exp)
+
+    def test_get_formats_exportable_formats(self):
+        obs = self.pm._exportable
+        exp = self.pm.get_formats(include_all=False, importable=False,
+                                  exportable=True, canonical_format=False)
+
+        self.assertEqual(obs, exp)
+
+    def test_get_formats_canonical_formats(self):
+        obs = self.pm._canonical_formats
+        exp = self.pm.get_formats(include_all=False, importable=False,
+                                  exportable=False, canonical_format=True)
+
+        self.assertEqual(obs, exp)
```

Comment:
These tests kind of "beg their own question", so I would drop them and think about how to test the implementation of *where* these results come from instead (i.e., why should we expect `self.pm._importable/_exportable/etc` to be correct). Also, those properties are currently being worked on, so this is all likely to change anyhow.

I think we've discussed this, so it's probably on someones list already, but just wanted to mention it again here.